.github/workflows/android.yml

```diff
@@ -17,8 +20,15 @@ jobs:
       uses: actions/setup-java@v1
       with:
         java-version: 11
-    - name: Copy CI gradle.properties
-      run: mkdir -p ~/.gradle ; cp .github/ci-gradle.properties ~/.gradle/gradle.properties
+  #  - name: Copy CI gradle.properties
```

Comment:
this may be required ?

Reply:
Right now there is no such file as ci-gradle.properties to copy the contents. The default gradle settings are sufficient. So, I commented out this part of the code.